vuln/npm/431.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": 431,
+  "title": "Out-of-bounds Read",
+  "overview": "`npmconf` (and `npm` js api) allocate and write to disk uninitialized memory content when a typed number is passed as input on Node.js 4.x",
+  "created_at": "2018-02-27",
+  "updated_at": "2018-05-12",
+  "publish_date": "2018-05-12",
+  "author": "Сковорода Никита Андреевич (https://github.com/ChALkeR)",
+  "module_name": "npmconf",
+  "cves": [],
+  "vulnerable_versions": "<=2.1.2",
+  "patched_versions": ">=2.1.3",
+  "recommendation": "update npmconf to 2.1.3 or higher",
```

Comment:
… or consider switching to another config storage mechanism, as `npmconf` is deprecated and should not be used

Reply:
already merged but I'll make changes.